_posts/2017-12-05-progress.markdown

```diff
@@ -0,0 +1,9 @@
+---
+layout: default
+img: code-screen.jpg
+category: Progress
+title: We're Making Progress
+description: |
+---
+
+We are now well into development of the Project Electron transfer application. Beyond the function of enabeling the transfer of records and metadata, we have been building out the application's capabilities for archival appraisal and accessioning tasks, rights management, and use by donors to track their transfers. You can read more about project progress from our blog posts on behavior driven development and our [approach to quality assurance](http://blog.rockarch.org/?p=1832), [defining requirements](http://blog.rockarch.org/?p=1859) for the second phase of the project using user scenario mapping and process diagrams, [developing a data model](http://blog.rockarch.org/?p=1865), and [user interface design](http://blog.rockarch.org/?p=1923).
```

Comment:
Maybe also say something about the transfer app name, aka Aurora, and link to the repo?